### PR TITLE
chat2 feature flag - hide edit menu

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers I added a new feature flag for things we want off for the release. 
I'm leaving this off before the release so people dogfooding will only see the expected alpha.
let's use this for features we don't intend to go out right away 

This is currently only used to hide the edit menu in messages

(see split out sort props commit)

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
The commits 9e060dbaf7bb...812ce7044792 introduce visual changes on linux. <details><summary>:mag_right: 2 changed</summary>

 * Changed: **Popup-You_Valid** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484318883914-9e060dbaf7bb-812ce7044792-linux/Popup-You_Valid-9e060dbaf7bb.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484318883914-9e060dbaf7bb-812ce7044792-linux/Popup-You_Valid-812ce7044792.png)  
   Popup-You_Valid comparison
 * Changed: **Popup-You_Revoked** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484318883914-9e060dbaf7bb-812ce7044792-linux/Popup-You_Revoked-9e060dbaf7bb.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484318883914-9e060dbaf7bb-812ce7044792-linux/Popup-You_Revoked-812ce7044792.png)  
   Popup-You_Revoked comparison
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 8c84187091cb...27cf4e4b1fe6 introduce visual changes on linux. <details><summary>:mag_right: 2 changed</summary>

 * Changed: **Popup-You_Valid** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484337912459-8c84187091cb-27cf4e4b1fe6-linux/Popup-You_Valid-8c84187091cb.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484337912459-8c84187091cb-27cf4e4b1fe6-linux/Popup-You_Valid-27cf4e4b1fe6.png)  
   Popup-You_Valid comparison
 * Changed: **Popup-You_Revoked** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484337912459-8c84187091cb-27cf4e4b1fe6-linux/Popup-You_Revoked-8c84187091cb.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484337912459-8c84187091cb-27cf4e4b1fe6-linux/Popup-You_Revoked-27cf4e4b1fe6.png)  
   Popup-You_Revoked comparison
</summary>

File: shared/chat/conversation/messages/popup.desktop.js [hunk 1]
```diff
@@ -57,9 +58,15 @@ export const TextPopupMenu = ({message, onEditMessage, onDeleteMessage, onHidden
...
     items = [
-      {title: 'Edit', onClick: () => onEditMessage(message)},
-      {title: 'Delete', subTitle: 'Deletes for everyone', danger: true, onClick: () => onDeleteMessage(message)},
+      {onClick: () => onEditMessage(message), title: 'Edit'},
+      {danger: true, onClick: () => onDeleteMessage(message), subTitle: 'Deletes for everyone', title: 'Delete'},
     ]
+
+    if (!flags.chatAdminOnly) {
+      // remote edit
+      items.shift()
```
Comment (MarcoPolo) [inline shared/chat/conversation/messages/popup.desktop.js]: 🐑 Would be nicer to have this next to where Edit is placed, so if the list above is changed it's obvious how this affects it. i.e. doing
```
...(flags.chatAdminOnly ? {onClick: ... title: 'Edit'} : {})
```

**Comment by chrisnojima** [comment]
I originally had that but flow hated it

**Pull request merged**